crates/redis/src/lib.rs

```diff
@@ -87,16 +141,19 @@ impl TriggerExecutor for RedisTrigger {
         println!("Active Channels on {address}:");
         // Subscribe to channels
         for (channel, component) in self.channel_components.iter() {
-            tracing::info!("Subscribing component {component:?} to channel {channel:?}");
-            pubsub.subscribe(channel).await?;
-            println!("\t{channel}: [{}]", component.join(","));
+            if channel.contains(&address) {
+                let channel_name = channel.strip_prefix(&format!("{address}:"));
+                tracing::info!("Subscribing component {component:?} to channel {channel_name:?}");
+                pubsub.subscribe(channel_name).await?;
+                println!("\t{channel}: [{}]", component.join(","));
```

Comment:
the channel may be ambiguous - consider disamiguating if so (or if override? not sure what's best)

Reply:
This should be fixed now as it prints the `{server_address}:{channel}`